src/main/java/edu/kis/vh/nursery/factory/ArrayStackRhymersFactory.java

```diff
@@ -0,0 +1,26 @@
+package edu.kis.vh.nursery.factory;
+
+import edu.kis.vh.nursery.CountingOutRhymer;
+import edu.kis.vh.nursery.FIFORhymer;
+import edu.kis.vh.nursery.HanoiRhymer;
+
+public class ArrayStackRhymersFactory implements RhymersFactory {
+    @Override
+    public CountingOutRhymer getStandardRhymer() {
+        return new CountingOutRhymer();
+    }
+
+    public CountingOutRhymer getFalseRhymer() {
+        return new CountingOutRhymer();
```

Comment:
PLEASE FIX: getFalseRhymer should use different IntContainer than other factory methods.